pkg/virt-handler/devices/bridge.go

```diff
@@ -17,219 +16,23 @@ type HostBridge struct {
 func (HostBridge) Setup(vmi *v1.VirtualMachineInstance, hostNamespaces *isolation.IsolationResult, podNamespaces *isolation.IsolationResult) error {
 	for i, net := range vmi.Spec.Networks {
 		if net.HostBridge != nil {
-			podns, err := ns.GetNS(podNamespaces.NetNamespace())
-			if err != nil {
-				return err
+			c := exec.Command("virt-handler",
```

Comment:
This is creating the process in the same namespace that the virt-handler  parent is already in. I'm not sure how this helps us.